lambda/handler.js

```diff
@@ -1,7 +1,11 @@
 // Contains test functions right now
+///added headers to avoid CORS issues for now whil in dev
+///added methods for new queries
 
 'use strict';
 const shakespeare = require('./queryShakespeare')
+const github = require('./queryGitHub')
```

Comment:
You've named this import github, but you used gitHub in the query

Reply:
done 👍